pkg/tracing/tracing_test.go

```diff
@@ -113,6 +114,17 @@ func TestPropagateSpan(t *testing.T) {
 	}
 }
 
+func TestPropagateSpanError(t *testing.T) {
+	tracer := NewTracer(mocktracer.New())
+	tracer.inject = func(sm ot.SpanContext, format interface{}, carrier interface{}) error {
+		return errors.New("error")
```

Comment:
This is a callback function that needs to return an error, so I think the code is right as written.